contracts/marketplace/src/execute.rs

```diff
@@ -43,15 +43,11 @@ pub fn instantiate(
     };
     SUDO_PARAMS.save(deps.storage, &params)?;
 
-    let mut res = Response::new().add_attribute("action", "instantiate");
-
     if let Some(hook) = msg.ask_filled_hook {
         ASK_FILLED_HOOKS.add_hook(deps.storage, deps.api.addr_validate(&hook)?)?;
-        res = res.add_attribute("action", "add_ask_filled_hook");
-        res = res.add_attribute("hook", hook);
     }
 
-    Ok(res)
+    Ok(Response::new())
```

Comment:
Do we want any attributes here?

Reply:
An `instantiate` event is created by default on instantiation that includes the contract address: https://github.com/CosmWasm/wasmd/blob/main/EVENTS.md#standard-events-in-xwasm. I can't think of anything more the indexer would need here... can you?

Reply:
Nope, resolving.